iphone/titanium.xcconfig

```diff
@@ -10,7 +10,7 @@ TITANIUM_SDK_VERSION = 6.0.3.GA
 //
 // THESE SHOULD BE OK GENERALLY AS-IS
 //
-TITANIUM_SDK = /Users/cavazos/Library/Application Support/Titanium/mobilesdk/osx/6.0.3.GA
+TITANIUM_SDK = ~/Library/Application Support/Titanium/mobilesdk/osx/$(TITANIUM_SDK_VERSION)
```

Comment:
Also this is kind of annoying to get out of the box from the `appc new` template. Is that something that can be changed on that end?